projects/python-client/tests/conftest.py

```diff
@@ -1,15 +1,31 @@
+import pytest
+
+from datapane.client import config
 from datapane.common import DPMode, _setup_dp_logging, set_dp_mode
 
-# run this at global level as want debug logging for all tests
-set_dp_mode(DPMode.SCRIPT)
-_setup_dp_logging(verbosity=2)
 
+@pytest.fixture(autouse=True)
+def dp_setup(request, monkeypatch, tmp_path):
+    """
+    Set up the common environment for each test
+
+    Skip optional init tasks with::
+
+        @pytest.mark.skip_init
+    """
+    print("dp_setup")
```

Comment:
stray debug print

Reply:
I'll remove - there's a print in the dp_login fixture too, I'll remove those while I'm at it